modulemd/v2/tests/ModulemdTests/modulestream.py

```diff
@@ -1036,7 +1036,26 @@ def test_depends_on_stream(self):
                     'platform', 'f28'), False)
 
             self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
-            self.assertEqual(stream.depends_on_stream('base', 'f30'), False)
+            self.assertEqual(
+                stream.build_depends_on_stream(
+                    'base', 'f30'), False)
+
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict that you didn't fix here.

Reply:
Fixed it : 
[commit_link](https://github.com/fedora-modularity/libmodulemd/pull/210/commits/91b17b6c7662fc1d1ba84c041c971b7af1479ece)